CodeSniffer/File.php

```diff
@@ -2709,8 +2709,10 @@ public function getDeclarationName($stackPtr)
     public function isAnonymousFunction($stackPtr)
     {
         $tokenCode = $this->_tokens[$stackPtr]['code'];
-        if ($tokenCode !== T_FUNCTION) {
```

Comment:
AFAICS you could simplify this a bit by leaving the original code as is and just adding the following snippet above it:
```php
if ($tokenCode === T_CLOSURE) {
    return true;
}
```